src/BatteryStateOfChargeService.cpp

```diff
@@ -26,15 +26,21 @@
  */
 
 #include "BatteryStateOfChargeService.h"
+#include <IOSTREAM>
+#include <QTextStream>
+#include <QDebug>
+using std::cout; using std::endl;
```

Comment:
This can just be 
```c++
using namespace std;
```

But overall, the `using` directive is frowned upon on our team. Explicitly stating `std::endl` is easier to understand.